libs/utils/trace.py

```diff
@@ -173,14 +163,9 @@ def setXTimeRange(self, t_min=None, t_max=None):
         :param t_max: upper bound
         :type t_max: int or float
         """
-        if t_min is None:
-            self.x_min = 0
-        else:
-            self.x_min = t_min
-        if t_max is None:
-            self.x_max = self.time_range
-        else:
-            self.x_max = t_max
+        self.x_min = t_min if t_min else self.start_time
+        self.x_max = t_max if t_max else self.start_time + self.time_range
```

Comment:
I think you need `is not None`s in here, otherwise you can't set `t_min=0`, right?

Reply:
You're right, good catch.